document-store/src/main/java/org/hypertrace/core/documentstore/postgres/utils/PostgresUtils.java

```diff
@@ -570,5 +570,8 @@ public static void enrichPreparedStatementWithParams(
                 log.error("SQLException setting Param. key: {}, value: {}", k, v);
               }
             });
+    if (log.isDebugEnabled()) {
+      log.debug("Executing statement - preparedStatement: {}", preparedStatement);
```

Comment:
`preparedStatement` has toString, right?

Reply:
Already checked this part. It works fine.